CONTRIBUTORS

```diff
@@ -187,7 +187,7 @@ Christian Donat <https://github.com/cdonat2>
 Asuka Minato <https://asukaminato.eu.org>
 Dillon Baldwin <https://github.com/DillBal>
 Voczi <https://github.com/voczi>
-Ben Nguyen <105088397+bpnguyen107@users.noreply.github.com>    
+Ben Nguyen <105088397+bpnguyen107@users.noreply.github.com>     
```

Comment:
Are you finding you need to do this every time?

Reply:
I realized the check kept failing because I never changed my email in git config to match that email 😅 It should be good moving forward!